crowbar_framework/config/locales/crowbar/en.yml

```diff
@@ -541,6 +541,29 @@ en:
       active: 'Active'
       active_repo_tooltip: 'Managed nodes will be automatically configured to use repositories only if they are marked as active'
 
+  installer:
+    status:
+      installation_failed: 'Installation failed. Please check the log at /var/log/crowbar/install.log'
+    index:
+      installation_header: 'Installing Crowbar'
```

Comment:
Can we replace "Crowbar" with some string that comes from /opt/dell/crowbar_framework/config/branding.yml? That will make branding easier.